project1/application.py

```diff
@@ -82,3 +83,52 @@ def admin():
     users = Users.query.order_by(Users.timestamp).all()
     return render_template("users.html", users = users)
 
+@app.route("/book/<ISBN>", methods = ["GET"])
+def book_details(ISBN):
+    name = session["name"]
+    # ISBN = "1416949658"
+    if request.method == "GET":
+        isb = Review.query.filter_by(ISBN_No = ISBN).all()
+        rev = Review.query.filter(and_(Review.name == name, Review.ISBN_No == ISBN)).first()
+        if rev and rev.flag == 2 :
+            return render_template("submit.html", sub_flag = 1, rate = rev.review_rate, comm = rev.review_description, isb = isb)
```

Comment:
Is submit.html the page in which the reviews will be available?

Reply:
changes the name to bookpage.html

Reply:
changed*